grocery_story.R

```diff
@@ -1,6 +1,6 @@
 ## collect the variables, and pick one at random
 
-people <- c("Jennifer", "Albert", "Tiago", "Oisin", "Miriam", "Linda", "Emmy", "Daniel")
+people <- c("Jennifer", "Albert", "Tiago", "Oisin", "Miriam", "Linda", "Emmy", "Daniel", "Yeon")
```

Comment:
```suggestion
people <- c("Jennifer", "Albert", "Tiago", "Oisin", "Linda", "Emmy", "Daniel", "Yeon")
```
Miriam is not part of our workshop, so let's take her out right away!